config/services.php

```diff
@@ -33,7 +35,13 @@
             service('.zenstruck_foundry.story_registry'),
             service('.zenstruck_foundry.persistence_manager')->nullOnInvalid(),
             service('event_dispatcher'),
+            param('.zenstruck_foundry.validation_enabled'),
+            abstract_arg('validation_available'),
         ])
         ->public()
+
+        ->set('.zenstruck_foundry.validation_listener', ValidationListener::class)
+        ->args([service('validator')])
+        ->tag('kernel.event_listener', ['event' => AfterInstantiate::class, 'method' => '__invoke'])
```

Comment:
I was wondering if I should listen `AfterPersist` event, and trigger validation on this event for persisting factories, but it feels wrong to validate after persisting